### PR TITLE
Observation updates

### PR DESCRIPTION
**Pull request by svburke**
Jira Ticket: [PXP-xxxx](https://ctds-planx.atlassian.net/browse/PXP-xxxx)
- [ ] Remove this line if you've changed the title to (PXP-xxxx): <title>

### New Features
Updated descriptions for onset_unk and deceased_date.
Added properties cvd_yn and res_zip
Updated type from boolean to enum for pos_spec_unk, pos_spec_na and onset_unk

### CODE REVIEW

**Review by xritter1 — APPROVED** [review]

**Review by paulineribeyre — DISMISSED** [review]

File: gdcdictionary/schemas/observation.yaml [hunk 1]
```diff
@@ -238,9 +240,10 @@ properties:
 
   onset_unk:
     description: >
-      If symptomatic, onset date - unknown
-    type: boolean
-
+      Check "Unknown", if symptomatic but onset date is unknown.
```
Comment (gkuffel) [inline gdcdictionary/schemas/observation.yaml:243]: I think I'm just being picky, but not sure a "," is needed, it reads like Check "Unknown". But only if....

Comment (svburke) [reply gdcdictionary/schemas/observation.yaml]: Resolved

**Review by xritter1 — APPROVED** [review]
LGTM

**Review by qiongl-CTDS — APPROVED** [review]
LGTM

**Review by gkuffel — APPROVED** [review]

**Pull request merged**